src/video-sync-manager.ts

```diff
@@ -117,16 +117,21 @@ export class VideoSyncManager {
     const synchDelayThresholdNegative = -0.03;
     const maxGap = 4;
     const seekAhead = 0.25; // s
+    //on safari without it the video get stuck on every rate change
+    // @ts-ignore
+    const synchDelayDefaultThreshold = this._secondaryPlayer._localPlayer._env.browser.name == "Safari" ? 0.5 : 0;
```

Comment:
why value is exact 0.5?

Reply:
this is the lowest value that I found that fix the issue. with less then 0.5  there are still lots of problems in the video